src/status_im2/events.cljs

```diff
@@ -102,3 +104,65 @@
                                                                     {:logout? false}])
                                                 (re-frame/dispatch [:get-keycard-banner-preference]))}
             (initialize-app-db)))
+
+;;;; ACCOUNT ADDRESS GENERATION
+(defn normalize-derived-data-keys
```

Comment:
we should have separate namespace for these events

Reply:
Same as before, these changes were reverted in 3dc2cea849b74a2f8ba5bfa08c87e09f2d17c7f9